pkg/master/master.go

```diff
@@ -96,6 +96,12 @@ const (
 	DefaultEtcdPathPrefix = "/registry"
 )
 
+var (
+	// allAPIVersions records all AVIVersions that are supported at /apis, in the
+	// form of "group/version"
+	allAPIVersions []string
```

Comment:
Ah, I forgot to leave a note here. It's package level because we will need to add/delete elements in this variable when installing/deleting thirdparty resource, https://github.com/kubernetes/kubernetes/blob/master/pkg/master/master.go#L777